uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/models/calendar-feed-config.model.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-env amd */
+
+define([
+  'leave-absences/shared/modules/models',
+  'common/models/model',
+  'leave-absences/shared/apis/calendar-feed-config.api',
+  'leave-absences/shared/instances/calendar-feed-config.instance'
+], function (models) {
+  'use strict';
+
+  models.factory('CalendarFeedConfig', [
+    'Model', 'CalendarFeedConfigAPI', 'CalendarFeedConfigInstance',
+    function (Model, CalendarFeedConfigAPI, CalendarFeedConfigInstance) {
+      var methods = {
```

Comment:
instead of defining this variable here, you can do the following:

```js
function (Model, CalendarFeedConfigAPI, CalendarFeedConfigInstance) {
  return Model.extend({
    all: all
  });

  function all () {
    // ...
  }
}
```